tests/test_helpers_additional.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import annotations
+
+import os
+from pathlib import Path
+
+import pytest
+
+from poetry.utils import helpers
+
+
+def test_merge_dicts_merges_correctly() -> None:
+    d1 = {"a": 1, "b": {"x": 1}}
+    d2 = {"b": {"y": 2}, "c": 3}
+    helpers.merge_dicts(d1, d2)
+    assert d1 == {"a": 1, "b": {"x": 1, "y": 2}, "c": 3}
+
+
+def test_paths_csv_returns_comma_separated_string(tmp_path: Path) -> None:
+    paths = [tmp_path / "a.txt", tmp_path / "b.txt"]
+    result = helpers.paths_csv(paths)
+    assert "a.txt" in result and "b.txt" in result
+    assert "," in result
+
+
+def test_ensure_path_creates_directory(tmp_path: Path) -> None:
+    path = tmp_path / "new_dir"
+    path.mkdir()
+    result = helpers.ensure_path(path, is_directory=True)
+    assert result.exists()
+    assert result.is_dir()
+
+
+def test_is_dir_writable_returns_true(tmp_path: Path) -> None:
+    assert helpers.is_dir_writable(tmp_path) is True
+
+
+def test_pluralize_singular_and_plural() -> None:
+    # Actual behavior: returns the word only, not "1 file"
+    assert helpers.pluralize(1, "file") == "file"
+    assert helpers.pluralize(3, "file") == "files"
+
+
+def test_get_file_hash(tmp_path: Path) -> None:
```

Comment:
Here, it probably makes more sense to write a file (bytes) and check for the fixed hash.